src/NServiceBus.Transport.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -30,14 +30,13 @@ sealed class MessagePump : IPushMessages, IDisposable
         PushSettings settings;
         CriticalError criticalError;
         MessagePumpConnectionFailedCircuitBreaker circuitBreaker;
-        TaskScheduler exclusiveScheduler;
 
         // Start
         int maxConcurrency;
-        SemaphoreSlim semaphore;
+        long numberOfExecutingReceives;
```

Comment:
Hmm, I think I like `numberOfMessagesBeingProcessed` better

Reply:
Happy to do the change. Will also remove the comment